ddl/ddl.go

```diff
@@ -249,8 +249,8 @@ type DDL interface {
 	RecoverTable(ctx sessionctx.Context, tbInfo *model.TableInfo, schemaID, autoID, dropJobID int64, snapshotTS uint64) (err error)
 	DropView(ctx sessionctx.Context, tableIdent ast.Ident) (err error)
 	CreateIndex(ctx sessionctx.Context, tableIdent ast.Ident, unique bool, indexName model.CIStr,
-		columnNames []*ast.IndexColName, indexOption *ast.IndexOption) error
-	DropIndex(ctx sessionctx.Context, tableIdent ast.Ident, indexName model.CIStr) error
+		columnNames []*ast.IndexColName, indexOption *ast.IndexOption, ifNotExists bool) error
```

Comment:
Change this interface does not break CI?

Reply:
No, all tests passed.

Reply:
That's weird ... I doubt that some cases are not run in our CI

Reply:
all usages of `DropIndex` also updated.